### ISSUE TITLE
Reihenfolge der Kategorien links zufällig

### ISSUE THREAD

**Issue by widominik** [kjoscha/innoMap#15]
Die Kategorien auf der linken Seite werden mehr oder weniger zufällig aufgestellt. Es müsste eine Möglichkeit geben, diese für jede Innolandkarte selbst anzuordnen.

**Comment by kjoscha** [comment]
achso, solche Systemeffekte haben dann keine Kategorien?

Ist das Beispiel online? Bräuchte dann mal den Link und Zugangsdaten (Email)

**Comment by widominik** [comment]
Doch, sie sind in Kategorien eingeordnet, werden aber außerhalb der sonstigen Pins angezeigt. Man kann alle Navilayer ausgeschaltet haben, aber in den Systemeffekten die Pins seperat ein- und ausschalten.
